pyproject.toml

```diff
@@ -18,7 +18,7 @@ torchkbnufft-cpu = ["torchkbnufft", "cupy-cuda12x"]
 torchkbnufft-gpu = ["torchkbnufft", "cupy-cuda12x"]
 
 cufinufft = ["cufinufft>=2.4.0", "cupy-cuda12x"]
-tensorflow = ["tensorflow-mri==0.21.0", "tensorflow-probability==0.17.0", "tensorflow-io==0.27.0", "matplotlib==3.7"]
+tensorflow = ["tensorflow-mri>=0.22.0"]
```

Comment:
Did you test this? I remember this versioning was very specific to avoid issues

Reply:
Not really, but tensorflow-mri manages its own dependency internally. Its also removes the pinning on matplotlib that was causing issue with numpy 2.0